redis.conf

```diff
@@ -573,7 +573,7 @@ replica-serve-stale-data yes
 # may also cause problems if clients are writing to it because of a
 # misconfiguration.
 #
-# Since Redis 2.6 by default replicas are read-only.
```

Comment:
Same as above, there is no Valkey 2.6

Reply:
Do we have an agreed approach to versioning and referring to the history of our code (e.g., Redis 2.6)?